tests/integration/jinad/test_simple_hub_pods/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.3"
```

Comment:
This test does not work, and not sure if we can make it work (I think we would need to pass parameters for special cases where docker pods are used in a docker-compose).

So the first point is:

- We need to expose `/var/run/docker.sock:/var/run/docker.sock` which allows the docker containers created via the docker-compose to communicate with the `docker daemon`. (Thus, ContainerPea can start peas in Pods via docker SDK). First thing to notice. These new docker containers won't leave inside the docker images hidden from the outside, they are docker containers in the same host and visible via host.

- The problem is that right now this new docker is using network mode host, while the containers started by docker-compose have their own bridge network. I haven't been able to successfully connect from the inner `bridge network` to the new dockers in host mode.

- As a test exercise I started some new docker containers attached to this new `bridge` network, and I could communicate to the other containers.  (This could lead to the conclusion that we could have the new `Peas in containers` attached to this network. If we do so, these containers will be assigned a new IP Address that we can't control and the rest of the pods won't be able to rediscover them).

Reply:
I think it is not necessarily a problem not to enable this specific usecase, the problem I see is that it does not let us test the `ContainerPea` in `remote` configuration.